dunning/dunning/doctype/dunning_type/dunning_type.json

```diff
@@ -0,0 +1,83 @@
+{
+ "autoname": "field:dunning_type",
+ "creation": "2019-12-04 04:59:08.003664",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "dunning_type",
+  "dunning_fee",
+  "interest_rate"
+ ],
+ "fields": [
+  {
+   "fieldname": "dunning_fee",
+   "fieldtype": "Currency",
+   "in_list_view": 1,
+   "label": "Dunning Fee",
+   "reqd": 1
+  },
+  {
+   "fieldname": "interest_rate",
+   "fieldtype": "Float",
+   "in_list_view": 1,
+   "label": "Interest Rate",
+   "reqd": 1
+  },
+  {
+   "fieldname": "dunning_type",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Name",
```

Comment:
Please use "Title" or "Dunning Type" as the label. "Name" usually refers to the unique ID.

Reply:
Named it as Dunning Type